gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -362,6 +362,16 @@ else if (leaseValidityStatus == 2) {
     }
   }
 
+  /*
+    Determines if a lease can be acquired for the given flow. A lease is acquirable if
+    no existing lease record exists in arbiter table or the record is older then epsilon time
+   */
```

Comment:
probably no need for this comment here in the impl, but if you want one, bring it into line w/ the orig from the interface

Reply:
makes sense, since javadoc already there for the interface, removed comment from here